docs/source/conf.py

```diff
@@ -77,6 +83,11 @@
 .. |marker_2| image:: /_static/images/marker_2.png
     :scale: 80%
 
+.. # define a hard space useful for literal-blocks that has empty content.
+.. |space| raw:: html
```

Comment:
I need to check how this affects to final layout...

Reply:
I did not notice any significant blank space.
Maybe I was looking at the wrong place.